controllers/food-controllers.js

```diff
@@ -170,7 +172,7 @@ const getMealsCalories = async (req, res, next) => {
 };
 const getTotalDayCalories = async (req, res, next) => {
   try {
-    const { email } = req.session.user;
+    const { email } = await User.findById(req.userData.userId);
```

Comment:
Some of the methods uses `findOne` and here i see `findById`, i think we should be more consistent, use either one but only it